src/Service.Host/client/src/components/Root.js

```diff
@@ -106,43 +110,41 @@ const Root = ({ store }) => (
                                     component={Tariff}
                                 />
                                 <Route exact path="/products/maint/tariffs" component={Tariffs} />
-                            </Switch>
```

Comment:
I've removed the <Switch> components in the Root component as Switch elements cannot be nested. These can be safely removed as we only ever return a single component for each route and are always using exact matches.